src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -159,12 +159,34 @@ public static Tag parseTag(String tag) throws ParseException {
     public static LocalDateTime parseAppointmentDateTime(String dateTime) throws ParseException {
         requireNonNull(dateTime);
         String trimmedDateTime = dateTime.trim();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+        String[] dateTimeInfo = trimmedDateTime.split(" ");
+
+        if (dateTimeInfo.length != 2) {
+            throw new ParseException("Please ensure both Appointment date and time are entered in "
+                    + "dd-MM-yyyy HH:mm format.");
+        }
+
+        String dateInput = dateTimeInfo[0];
+        String timeInput = dateTimeInfo[1];
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```

Comment:
One more thing is to maybe make this check stricter!
`DateTimeFormatter dateFormat =
                DateTimeFormatter.ofPattern("dd-MM-uuuu").withResolverStyle(ResolverStyle.STRICT);`